VERSION

```diff
@@ -0,0 +1 @@
+1.7.2
```

Comment:
I like versioning the Docker image, and keeping the image tag in sync with the library version.

Reply:
#46 addressed that with an option in Drone: https://github.com/fsouza/fake-gcs-server/pull/46/files#diff-3216dfff0ed3e301453e6799e8c367e2R71

We can probably reuse it and get rid of the ``VERSION`` file (having the VERSION file gets tricky as we'd need to remember to always update it before tagging, with a CI it's automatic).

(I can do the drone integration part)